Firebase/Core/FIRApp.m

```diff
@@ -89,6 +89,8 @@ @interface FIRApp ()
 
 @property(nonatomic) BOOL alreadySentDeleteNotification;
 
+@property(nonatomic) BOOL alreadyOutputDataCollectionFlag;
```

Comment:
I think this may be unnecessary since some extra debug logging wouldn't hurt. No need to add an extra property to keep track of logging state.

Reply:
That was also my thought originally, but this would cause the flag to be output every time an SDK calls `sendLogsWithServiceName:version:error:`, which seems like it might be a little excessive.

I put the property behind an `#ifdef` so it shouldn't affect release builds.

Reply:
Ahh good insight! Thanks.